src/Alpha_complex/include/gudhi/Incremental_delaunay.h

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// As an exception, if you have a commercial license for the CGAL package Triangulation,
+// you may use this file under the same conditions.
+//
+// Author(s)    : Michael Kerber, Marc Glisse
+
+#include <CGAL/Delaunay_triangulation.h>
+#include <iterator>
+#include <vector>
+
+namespace Gudhi {
+
+/** \ingroup alpha_complex
+ *
+ * This function constructs the incremental Delaunay complex, as defined in \cite incremental-delaunay .
+ * Given a list of points (p1, ..., pn), the incremental Delaunay complex contains the Delaunay triangulation
+ * of (p1, ..., pk) for all k≤n. For any simplex s of the triangulation of (p1, ..., pk) that is not in the
+ * triangulation of (p1, ..., pn), the complex also contains a simplex formed with the same points as s plus the
+ * first pj that conflicts with s (is inside its circumsphere). For points in R^d, the complex usually has
```

Comment:
For a better LaTeX render:
```suggestion
* first pj that conflicts with s (is inside its circumsphere). For points in \f$\mathbb{R}^d\f$, the complex usually has
```